akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -281,7 +281,9 @@ object ActorMaterializerSettings {
    * Prefer using either config for defaults or attributes for per-stream config.
    * See migration guide for details https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html"
    */
-  @silent("deprecated")
+  @deprecated(
+    "Use config or attributes to configure the materializer. See migration guide for details https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html",
```

Comment:
maybe such link should use `2.6` instead of `current`?

Reply:
Hmm, yes, maybe that is better. I'll change.